### PR TITLE
KAFKA-8907; Return topic configs in CreateTopics response (KIP-525)

### PR DESCRIPTION
**Pull request by rajinisivaram**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by rajinisivaram** [comment]
retest this please

**Comment by rajinisivaram** [comment]
retest this please

File: clients/src/main/java/org/apache/kafka/clients/admin/CreateTopicsResult.java [hunk 1]
```diff
@@ -40,13 +44,94 @@
      * topic creations.
      */
     public Map<String, KafkaFuture<Void>> values() {
-        return futures;
+        return futures.entrySet().stream()
+                .collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue().thenApply(v -> (Void) null)));
     }
 
-    /**
+    /**TOPIC
```
Comment (omkreddy) [inline clients/src/main/java/org/apache/kafka/clients/admin/CreateTopicsResult.java]: unintended change?

**Review by omkreddy — APPROVED** [review]
@rajinisivaram Thanks for the PR.  LGTM.

File: clients/src/main/java/org/apache/kafka/clients/admin/CreateTopicsResult.java (hunk 1 above)
Comment (rajinisivaram) [reply clients/src/main/java/org/apache/kafka/clients/admin/CreateTopicsResult.java]: Oops, removed.

**Comment by rajinisivaram** [comment]
@omkreddy Thanks for the review, merging to trunk.

**Pull request merged**